server/services/omnichannel-voip/service.ts

```diff
@@ -32,11 +33,10 @@ export class OmnichannelVoipService extends ServiceClass implements IOmnichannel
 		this.logger = new Logger('OmnichannelVoipService');
 	}
 
-	private normalizeAgent(agentId: string): Record<string, string> | Record<string, boolean> | undefined {
+	private normalizeAgent(agentId: string): ILivechatAgent | { hiddenInfo: true } | undefined {
 		if (!agentId) {
 			return;
 		}
-
```

Comment:
@pierre-lehnen-rc I have fixed this. But I would like to mention that there is a semantic ambiguity here. Livechat is not a Voip. So returning ILivechatAgent introduces some semantic ambiguity. We have a similar case in for visitors. We are using all the APIs of LiveChat Visitor here. We may have to revisit this.